.github/workflows/samples_flows_chat_chatwithpdf_chatwithpdfazure.yml

```diff
@@ -9,7 +9,7 @@ on:
     - cron: "44 20 * * *" # Every day starting at 4:44 BJT
   pull_request:
     branches: [ main ]
-    paths: [ examples/flows/chat/chat-with-pdf/**, examples/*requirements.txt, .github/workflows/samples_flows_chat_chatwithpdf_chatwithpdfazure.yml ]
+    paths: [ examples/flows/chat/chat-with-pdf/**, examples/*requirements.txt, .github/workflows/samples_flows_chat_chatwithpdf_chatwithpdfazure.yml, '!examples/flows/integrations/**' ]
```

Comment:
why do we need to add this to here?

Reply:
this is the auto-gen result, maybe we can regard it as an unnecessary rule and won't work; I will probably fix this for the work item that generates rules via config.